mod/engine-primitives/pkg/engine-primitives/attributes_test.go

```diff
@@ -29,76 +29,97 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-type Withdrawal struct{}
+type testWithdrawal struct{}
 
-func TestPayloadAttributes(t *testing.T) {
```

Comment:
this comes from merging https://github.com/berachain/beacon-kit/pull/2068 into this PR